operations/observability/mixins/workspace/rules/components/workspaces/alerts.libsonnet

```diff
@@ -43,6 +43,20 @@
               ) without(phase) > 5
             |||,
           },
+          {
+            alert: 'GitpodWorkspaceHighFailureRate',
+            labels: {
+              severity: 'critical',
+            },
+            annotations: {
+              runbook_url: '',
```

Comment:
I think we need runbook for all our alerts, no?

Reply:
Hi @princerachit I shared a response yesterday, do you need anything else from the team?

Reply:
Nope, I have updated the runbook url.